content/blog/blog-2020-07-01-kudo-dependencies.md

```diff
@@ -0,0 +1,95 @@
+---
+date: 2020-07-01
+---
+
+# KUDO Dependencies
+
+One of the most requested features (internally and externally) has been operator dependencies. We've experienced this pain ourselves: our own [Kafka](https://github.com/kudobuilder/operators/tree/master/repository/kafka) operator requires the user to manually [install Zookeeper](https://github.com/kudobuilder/operators/blob/master/repository/kafka/docs/v1.3/install.md) as a prerequisite. This sounds simple enough but can get out of hand quickly as our [Flink demo](https://github.com/kudobuilder/operators/tree/master/repository/flink/docs/demo/financial-fraud) shows.
```

Comment:
Nitpick, or maybe not a nitpick in CNCF context: Who is "us" when you mention "our own" operator here? IMO, we have to be careful to distinguish the ownership here: The Kafka operator is an [open-source project by D2iQ](https://github.com/mesosphere/kudo-kafka-operator) published in the community repository of KUDO. Even though the same team at D2iQ works on both projects, we have to distinguish in the external ownership.

Reply:
That's an excellent point! I'll change the wording